docs/src/ref/mcmc.md

```diff
@@ -201,6 +201,207 @@ new_trace = permute_move(trace, 2)
+The body of the function can contain almost arbitrary Julia code.
+However, reads from ``(t, u)`` and writes to ``(t', u')`` use specific macros.
+Some of these macros can only be used with either discrete or continuous random choices, respectively:
+
+- `@read_discrete_from_model(addr)`: Read the discrete value from the input model choice map (``t``) at the given address.
+
+- `@write_discrete_to_model(addr, value)`: Write a discrete value to the output model choice map (``t'``) at the given address.
+
+- `@read_discrete_from_proposal(addr)`: Read the discrete value from the input proposal choice map (``u``) at the given address.
+
+- `@write_discrete_to_proposal(addr, value)`: Write a discrete value to the output proposal choice map (``u'``) at the given address.
+
+- `@read_continuous_from_model(addr)`: Read the continuous value from the input model choice map (``t``) at the given address.
+
+- `@write_continuous_to_model(addr, value)`: Write a continuous value to the output model choice map (``t'``) at the given address.
+
+- `@read_continuous_from_proposal(addr)`: Read the continuous value from the input proposal choice map (``u``) at the given address.
+
+- `@write_continuous_to_proposal(addr, value)`: Write a continuous value to the output proposal choice map (``u'``) at the given address.
+
+Often involutions directly copy the value from one address in the input ``(t, u)`` to the output ``(t', u')``.
+In these cases, the implementation will be more efficient if explicit 'copy' commands are used instead:
+
+- `@copy_model_to_model(from_addr, to_addr)`: Copy the value (discrete or continuous) or an entire sub-map of choices under an address namespace from the input model choice map (``t``) to the output model choice map (``t'``).
+
+- `@copy_model_to_proposal(from_addr, to_addr)`: Copy the value (discrete or continuous) or an entire sub-map of choices under an address namespace from the input model choice map (``t``) to the output proposal choice map (``u'``).
+
+- `@copy_proposal_to_proposal(from_addr, to_addr)`: Copy the value (discrete or continuous) or an entire sub-map of choices under an address namespace from the input proposal choice map (``u``) to the output proposal choice map (``u'``).
+
+- `@copy_proposal_to_model(from_addr, to_addr)`: Copy the value (discrete or continuous) or an entire sub-map of choices under an address namespace from the input proposal choice map (``u``) to the output model choice map (``t'``).
+
+It is not necessary to explicitly copy values from the previous model choice map (``t``) to the new model choice map (``t'``) at the same address.
+These values will be copied automatically by the system.
+
+Finally, When reading a continuous value from the input model choice map that is expected to be implicitly copied by the system, the system will be more efficient if the following command is used instead of `@read_continuous_from_model`:
```

Comment:
@alex-lew For typos, you can make a Suggestion in your comment and then apply that Suggestion by point-and-click.  About to verify that what I said is true by doing so.
```suggestion
Finally, when reading a continuous value from the input model choice map that is expected to be implicitly copied by the system, the system will be more efficient if the following command is used instead of `@read_continuous_from_model`:
```